lib/plugins/after/metrics/ObjectStyle.js

```diff
@@ -0,0 +1,130 @@
+// jscs:disable disallowOperatorBeforeLineBreak
+
+'use strict';
+
+var assert = require('assert-plus');
+
+var getAuditPayload = require('./../getAuditPayload');
+var requestSerializer = require('./../../serializers/requestSerializer');
+var responseSerializer = require('./../../serializers/responseSerializer');
+
+var UrlStore = require('./UrlStore');
+var TimerStore = require('./TimerStore');
+var StatusCodeStore = require('./StatusCodeStore');
+
+var DEFAULT_MAX_REQUESTS = 1000;
+
+///--- API
+
+/**
+ * The HTTPMetrics capturer.  The options passed in requires the
```

Comment:
Not quite sure I understand the meaning of the second sentence. Would you mind rewording?